docs/dev/build_raspbian.md

```diff
@@ -43,9 +43,9 @@ git clone --recurse-submodules --single-branch --branch=master https://github.co
   via `pip3`, adding the following options:
    ```sh
    -DENABLE_PYTHON=ON \
-   -DPYTHON_EXECUTABLE=/usr/bin/python3.7 \
-   -DPYTHON_LIBRARY=/usr/lib/arm-linux-gnueabihf/libpython3.7m.so \
-   -DPYTHON_INCLUDE_DIR=/usr/include/python3.7
+   -DPYTHON_EXECUTABLE=/usr/bin/python3.8 \
+   -DPYTHON_LIBRARY=/usr/lib/arm-linux-gnueabihf/libpython3.8m.so \
```

Comment:
remove `m` suffix in other place in docs as well

Reply:
Done in https://github.com/openvinotoolkit/openvino/pull/19670/commits/eb4ec4338a562277b675eb6a208aa723f07b3b6a